tests/Controller/ControllerTestTrait.php

```diff
@@ -40,34 +40,47 @@ public function assertClientIsRedirectedTo(string $path, Client $client, $withSc
         );
     }
 
+    public function assertClientIsRedirectedToAuth()
+    {
+        $this->assertSame(
```

Comment:
It looks wrong to use a method that does not exist in the trait itself but anyway it was the case before